files/en-us/web/javascript/reference/global_objects/array/at/index.html

```diff
@@ -0,0 +1,121 @@
+---
+title: Array.prototype.at()
+slug: Web/JavaScript/Reference/Global_Objects/Array/at
+tags:
+  - Array
+  - JavaScript
+  - Method
+  - Prototype
+  - Reference
+  - polyfill
+  - at
+  - Experimental
+---
+<div>{{JSRef}}</div>
+
+<p class="summary">The <strong><code>at()</code></strong> method takes an integer value and returns the item at that index, allowing for negative integers which count back from the last item.</p>
+
+<div>{{EmbedInteractiveExample("pages/js/array-at.html","shorter")}}</div>
+
+<ul>
+  <li>If you need to find an element based on a provided testing function, use {{jsxref("Array.prototype.find()", "find()")}}.</li>
+  <li>If you need to find if a value <strong>exists</strong> in an array, use {{jsxref("Array.prototype.includes()")}}.</li>
+  <li>If you need the <strong>index</strong> of an element in the array, use {{jsxref("Array.indexOf", "indexOf()")}}.</li>
+</ul>
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre class="syntaxbox notranslate">var <var>item</var> = arr.at(index)</pre>
```

Comment:
I'd remove the variable assignment in the syntax section. This section is mostly meant to show the method's signature. (also someone will probably come a long and say "Don't use var!".

```suggestion
<pre class="syntaxbox notranslate">arr.at(index)</pre>
```

Reply:
Thanks @Elchi3 so just to clarify even if something is returned we don't tend to add var (or let or whatever stops peeps complaining 😆) - I've done this based on other examples.